ledger/catchpointtracker.go

```diff
@@ -69,10 +69,13 @@ const (
 
 	// CatchpointFileVersionV5 is the catchpoint file version that was used when the database schema was V0-V5.
 	CatchpointFileVersionV5 = uint64(0200)
-	// CatchpointFileVersionV6 is the catchpoint file version that is matching database schema V6.
+	// CatchpointFileVersionV6 is the catchpoint file version that is matching database schema V6-V9.
```

Comment:
This comments should be updated every time the version changes?
```suggestion
	// CatchpointFileVersionV6 is the catchpoint file version that is matching database schema since V6.
```